lib/generators/boring/sentry/install/install_generator.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Boring
+  module Sentry
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds Sentry to the app'
+
+      class_option :use_env_variable, type: :boolean, aliases: '-ev',
+                   desc: 'Use ENV variable for devise_jwt_secret_key. By default Rails credentials will be used.' 
+
+      class_option :breadcrumbs_logger, type: :array, aliases: '-bl',
+                   desc: 'Set the breadcrumbs logger. By default [:active_support_logger, :http_logger] will be used.'
+
+      def add_sentry_gems
+        say 'Adding Sentry gem', :green
+
+        gem 'sentry-ruby'
+        gem 'sentry-rails'
+      end
+
+      def configure_sentry_gem
+        say 'Configuring Sentry gem', :green
+
+        sentry_config = <<~RUBY
+          Sentry.init do |config|
+            config.dsn = #{sentry_dsn_key}
+            # enable performance monitoring
+            config.enable_tracing = true
+            # get breadcrumbs from logs
+            config.breadcrumbs_logger = #{breadcrumbs_logger_options}
+          end
+        RUBY
+
+        create_file 'config/initializers/sentry.rb', sentry_config
```

Comment:
Let's use template to do this since we are creating new file, you have already done similar thing in brakeman with github ci